x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -296,6 +310,27 @@ export class Simulator {
     return this.domNodes(`[data-test-subj="${selector}"]`);
   }
 
+  /**
+   * Given a `ReactWrapper`, returns a wrapper containing immediately following `dd` siblings.
+   * `subject` must contain just 1 element.
+   */
+  public descriptionDetails(subject: ReactWrapper): ReactWrapper {
```

Comment:
I couldn't find an easy way to find the related `dd` element from a `dt` element using enzyme's `ReactWrapper`. There does not seem to be any way to iterate siblings. This takes a wrapper that contains exactly one `dt` and returns its related `dd`'s.